stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -225,6 +225,12 @@ end
     end
 end
 
+@testset "Issue #30466" begin
+    @test norm([typemin(Int), typemin(Int)], Inf) == 9.223372036854776e18
+    @test norm([typemin(Int), typemin(Int)], -Inf) == 9.223372036854776e18
```

Comment:
Did we discuss this anywhere? I'm not sure we should allow negative `p` arguments so probably better to leave out this test.